includes/features/all_sites.feature

```diff
@@ -0,0 +1,53 @@
+Feature: Every page will be checked for the following regions, elements and functions
+  In order to ensure that the content on all sites exist,
+  As an end user
+  I want to check whether the content exists
+
+  @deploy
```

Comment:
@kmakitan , These scenarios can be run on any sites right? Then the tags should be @safe.